File: KP/revision/W2/CPP/TemperaturaTest.cpp
#include <iostream>
#include "Temperatura.hpp"

using namespace std;

int main(int args, char* Arg[]) {
    Temperatura *t1 = new Temperatura();
    Temperatura *t2 = new Temperatura(56.7);

    cout << "TEMPERATURA W CELSJUSZACH: " << t2->Celsjusz() << " W KELVINACH: " << t2->Kelvin() << " W FAHRENHAITACH: " << t2->Fahrenheit() << endl;

    t1->setK(273.15);
    cout << "TEMPERATURA W CELSJUSZACH: " << t1->Celsjusz() << " W KELVINACH: " << t1->Kelvin() << " W FAHRENHAITACH: " << t1->Fahrenheit() << endl;

    delete t1;
    delete t2;
}

File: KP/revision/W2/JAVA/TemperaturaTest.java
public class TemperaturaTest {
    public static void main(String Args[]) {
        Temperatura t = new Temperatura();
        Temperatura t2 = new Temperatura(33.4);

        System.out.println("Temperatura w Celsjuszach: " + t2.Cejsjusz() + " w Kelvnach: " + t2.Kelvin() + " w Fahrenhaitach: " + t2.Fahrenheit());
        t.setC(16.5);
        System.out.println("Temperatura w Celsjuszach: " + t.Cejsjusz() + " w Kelvnach: " + t.Kelvin() + " w Fahrenhaitach: " + t.Fahrenheit());
    }
}
